### PR TITLE
added logstash info

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _clients/logstash/send-to-opensearch.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Send events to Opensearch
+parent: Logstash
+nav_order: 220
+---
+
+# Send events to Opensearch
+
+You can send Logstash events to an Opensearch cluster and then visualize your events with Kibana.
```
Comment (VijayanB) [inline _clients/logstash/send-to-opensearch.md:10]: Kibana -> OpenSearch Dashboards

File: _clients/logstash/send-to-opensearch.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Send events to Opensearch
+parent: Logstash
+nav_order: 220
+---
+
+# Send events to Opensearch
```
Comment (VijayanB) [inline _clients/logstash/send-to-opensearch.md:8]: OpenSearch

File: _clients/logstash/send-to-opensearch.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+    `config/pipeline.conf` is a relative path to the `pipeline.conf` file. You can use an absolute path as well.
+
+3. Add a JSON object in the terminal:
+
+    ```json
+    { "amount": 10, "quantity": 2}
+    ```
+
+4. Open Opensearch and search for the processed event:
```
Comment (VijayanB) [inline _clients/logstash/send-to-opensearch.md:73]: Open Dev Tools

File: _clients/logstash/send-to-opensearch.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Send events to Opensearch
```
Comment (VijayanB) [inline _clients/logstash/send-to-opensearch.md:3]: Ship logs to OpenSearch

File: _clients/logstash/send-to-opensearch.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      opensearch {
+        hosts       => "https://localhost:9200"
+        user        => "admin"
+        password    => "admin"
+        index       => "logstash-logs-%{+YYYY.MM.dd}"
+        ssl_certificate_verification => false
+      }
+    }
+    ```
+
```
Comment (VijayanB) [inline _clients/logstash/send-to-opensearch.md:58]: Is it possible to explain the configuration in words?

File: _clients/logstash/index.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+## Install Logstash on MAC / Linux
+
+Make sure you have [Java Development Kit (JDK)](https://www.oracle.com/java/technologies/javase-downloads.html) version 8 or 11 installed.
+
+1. Download the Logstash tarball from [OpenSearch downloads](https://opensearch.org/downloads.html).
+
+2. Navigate to the downloaded folder in the terminal and extract the files:
+
+     ```bash
+     tar -zxvf logstash-7.13.2-darwin-x86_64.tar.gz
```
Comment (VijayanB) [inline _clients/logstash/index.md:61]: logstash-oss-with-opensearch-output-plugin-7.13.2-macos-x64.tar.gz

File: _clients/logstash/index.md [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+* `filter` parses and enriches the events in one way or the other. Logstash has a large collection of filter plugins that modify events and pass them on to an output. For example, a `grok` filter parses unstructured events into fields and a `mutate` filter changes fields. Filters are executed sequentially.
+* `output` ships the filtered events to one or more destinations. Logstash supports a wide range of output plugins for destinations like OpenSearch, TCP/UDP, emails, files, stdout, HTTP, Nagios, and so on.
+
+Both the input and output phases support codecs to process events as they enter or exit the pipeline.
+Some of the popular codecs are `json` and `multiline`. The `json` codec processes data that’s in JSON format and the `multiline` codec merges multiple line events into a single line.
+
+You can also write conditional statements within pipeline configurations to perform certain actions, if a certain criteria is met.
+
+## Install Logstash on MAC / Linux
```
Comment (VijayanB) [inline _clients/logstash/index.md:52]: For GA we will be supporting only Linux

**Comment by VijayanB** [comment]
Shall we add section on how to use Docker as well?

File: _clients/logstash/index.md (hunk 7 above)
Comment (ashwinkumar12345) [reply _clients/logstash/index.md:52]: Fixed.

**Comment by elfisher** [comment]
> Shall we add section on how to use Docker as well?

I think that makes sense, but let's track that on a separate issue unless @ashwinkumar12345 you want to use this issue to track that?

**Comment by ashwinkumar12345** [comment]
I can create a separate issue for this. Do we have a docker-compose.yml file for installing Logstash with Docker?

**Comment by elfisher** [comment]
> I can create a separate issue for this. Do we have a docker-compose.yml file for installing Logstash with Docker?

Not sure. @VijayanB do you have a compose file or Dockerfile we could use for this?

**Review by aetter — APPROVED** [review]
Approving so that you can merge whenever you have download location.

File: _clients/logstash/index.md [hunk 8]
```diff
@@ -0,0 +1,356 @@
...
+1. Create a Docker network:
+
+```
+docker network create test
+```
+
+1. Start OpenSearch with this network:
+
+```
+docker run -p 9200:9200 -p 9600:9600 --name opensearch --net mynet -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
```
Comment (VijayanB) [inline _clients/logstash/index.md:88]:
```suggestion
docker run -p 9200:9200 -p 9600:9600 --name opensearch --net test -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
```

File: _clients/logstash/index.md [hunk 9]
```diff
@@ -0,0 +1,356 @@
...
+```
+
+1. Start Logstash:
+
+```
+docker run -it --rm --name logstash1 --net mynet opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
```
Comment (VijayanB) [inline _clients/logstash/index.md:94]:
```suggestion
docker run -it --rm --name logstash --net test opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
```

File: _clients/logstash/index.md [hunk 10]
```diff
@@ -0,0 +1,356 @@
...
+  opensearch {
+    hosts => ["https://opensearch:9200"]
+    index => "opensearch-logstash-mac-x64-docker-%{+YYYY.MM.dd}"
```
Comment (VijayanB) [inline _clients/logstash/index.md:97]:
```suggestion
    index => "opensearch-logstash-docker-%{+YYYY.MM.dd}"
```

File: _clients/logstash/index.md [hunk 11]
```diff
@@ -85,16 +85,16 @@ docker network create test
...
 ```
-docker run -p 9200:9200 -p 9600:9600 --name opensearch --net mynet -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
+docker run -p 9200:9200 -p 9600:9600 --name opensearch --net test -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
 ```
 
 1. Start Logstash:
 
 ```
-docker run -it --rm --name logstash1 --net mynet opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
+docker run -it --rm --name logstash --net test opensearchstaging/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
```
Comment (VijayanB) [inline _clients/logstash/index.md:94]:
```suggestion
docker run -it --rm --name logstash --net test openserachproject/logstash-oss-with-opensearch-output-plugin:7.13.2 -e 'input { stdin { } } output {
```

**Pull request merged**